torchvision/csrc/io/decoder/video_sampler.cpp

```diff
@@ -65,6 +64,48 @@ int transformImage(
   if ((result = preparePlanes(outFormat, out, planes, lines)) < 0) {
     return result;
   }
+
+  // Here we're changing colourspace in order to maintain consistency with the
+  // GPU decoder Since SWSContext cannot be edited directly, we're borrowing
+  // pyAV approach from:
+  // https://github.com/PyAV-Org/PyAV/blob/9ac05d9ac902d71ecb2fe80f04dcae454008378c/av/video/reformatter.pyx#L141
+  // very sory for this, found no cleaner way of doing it
```

Comment:
I think this is just an FYI, perhaps we don't need the last statement to be checked-in.